dev/TreeView/TreeViewItem.xaml

```diff
@@ -30,25 +30,46 @@
                         contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Normal"/>
-                                <VisualState x:Name="PointerOver">
+                                <VisualState x:Name="Normal">
```

Comment:
The default visual state should not have any setters, as its styles should be applied to the default template.

Reply:
it's actually common practice to define normal state to make sure eveything resets cleanly and it makes it easier to read instead of going back to the default setters. @marksfoster any thoughts?

Reply:
This is also so that it stays consistent with the rest of the WinUI2 repo.

Reply:
I'm partial to empty "Normal" states but either is fine.